www/class/centreon-clapi/centreonHost.class.php

```diff
@@ -1313,6 +1316,7 @@ public function export($filterName = null)
                     "ehi_action_url",
                     "ehi_icon_image",
                     "ehi_icon_image_alt",
+                    "ehi_vrml_image",
```

Comment:
what is this for ?

Reply:
This is obsolete now but still present in DB schema